packages/fast-components-styles-msft/src/checkbox/index.ts

```diff
@@ -21,6 +21,7 @@ const styles: ComponentStyles<ICheckboxClassNameContract, IDesignSystem> = {
         boxSizing: "content-box",
         margin: "0",
         zIndex: "1",
+        alignSelf: "center",
```

Comment:
I think we want the parent to dictate this. Per our convo earlier, I have this in a branch with other fixes. I think the parent should set the alignment of the children to ensure that both are centered vertically with one another.